tests/plain-nodaily/test.toml

```
@@ -0,0 +1,56 @@
+# Test graphite_index with disable-daily-index = false
+[test]
+config_template = "carbon-clickhouse.conf.tpl"
+input = [ 
+    "test.host1.cpu.loadavg 10.2 1625478240",
+    "test.host1.cpu.loadavg 9.4 1625478300",
+]
+
+[[test.clickhouse]]
+version = "19.7"
```

Not sure why it's exactly this version, but LTS 19.14 looks a little bit better to me
```suggestion
version = "19.14"
```

https://repo.clickhouse.tech/deb/lts/main/